### DISCUSSION TITLE
Are there any weather  predictive data ? If not, the problem will be really HARD

### DISCUSSION THREAD

**Discussion by itachiCheng** [PaddlePaddle/PaddleSpatial#16] [BaiduKDDCup2022] [answered]
Are there any weather  predictive data ? If not, the problem will be really HARD

**Comment by itachiCheng** [comment]
I know in this field, the weather report data is also needed for modeling.

**Comment by zhoujingbo** [comment, marked as answer]
Hi,

The weather report data will not be provided.

In our settings, we aim to forecast the short-term (within 1~2 days) power generated by a wind farm without knowing the future weather conditions, which is more challenging but more applicable for the majority.  We treat how to use the weather report data to predict wind power as another line of this problem. 

Thanks for your understanding.

**Reply by itachiCheng** [reply]
OK, let me try.

**Reply by RobinHan24** [reply]
Hi organizers, The following sentence is directly quoted from the report you provided - 'we still privately hold data of several months to evaluate the submitted models by participants. ' Does it mean that in the evaluate dataset, meteorological data will not be available or available? And how many data fileds in in the future test set? If there are 13 fields in the future test set which is the same as train dataset, so the value of meteorological data will be NAN? Please tell us which columns are included in the future test set? Thank you very much.

**Reply by RobinHan24** [reply]
> OK, let me try.

So which columns are included in the future test set?  Thanks a lot.
